extensions/blocks/opentable/editor.scss

```diff
@@ -66,20 +73,32 @@
 		width: 100%;
 
 		.components-form-token-field__input-container {
-			width: 327px;
-			max-width: 100%;
+			width: 100%;
+			@media screen and (min-width: 480px) {
+				max-width: 327px;
+			}
 		}
 
 		.components-button.is-large {
-			height: 38px;
-			margin: 4px 0 0 0;
-			position: absolute;
-			right: 0;
+			@media screen and (min-width: 480px) {
+				margin: 4px 0 0 0;
+				position: absolute;
+				right: 0;
+				top: 4.4em;
```

Comment:
@scruffian I don't feel good about this, but it works. Have you got any thoughts? The idea is to position the button to stop it jumping around with the suggestion list, but it's not part of the `FormTokenField`s container. 

On mobile, I've made it centred beneath the field, which also isn't fantastic. We could potentially use `calc` to change the width of the input, but either way, it's all a bit hacky. 

Ideas are very welcome!

Reply:
I updated this to use flexbox which seems a bit more robust